src/components/common-components/button-component/button.component.tsx

```diff
@@ -6,7 +6,7 @@ export type ButtonOptions = Omit<ButtonProps, "variant"> & {
 };
 
 export const Button = React.forwardRef(
-  ({ variant = "primary", ...rest }: ButtonOptions, ref?: Ref<HTMLButtonElement>) => {
-    return <button {...rest} ref={ref} className={`btn btn-${variant}`} />;
+  ({ variant = "primary", className, ...rest }: ButtonOptions, ref?: Ref<HTMLButtonElement>) => {
```

Comment:
Nie wiem czy to zgodne ze sztuką css'a, ale wpadłem na taki pomysł, żeby przekazywać customowe style do tych przycisków.

Reply:
Moim zdaniem, ma sens, ponieważ czasami tego potrzebujemy. 
Jedyne co, ja bym przypisał domyślną wartość do `className`, ponieważ nie mamy gwarancji, że zawsze będą go podawać przez co może pojawić się `undefined` jako klasa. Nie jest to źle, ale trochę nieprzyjemnie moim zdaniem wygląda

Reply:
Słuszna uwaga, jednak wolałbym nie dodawać wartość domyślnej, a zamiast tego sprawdzać czy wartość nie jest undefined

Reply:
Z rozwiązaniem problemu przychodzi zaproponowana przez @maciekb05 libka. Jeśli coś jest undefined to nie zostanie uwzględnione :) 
![image](https://user-images.githubusercontent.com/23723122/101635164-e2519680-3a29-11eb-8728-06bc414ad934.png)